packages/kbn-optimizer/limits.yml

```diff
@@ -114,5 +114,5 @@ pageLoadAssetSize:
   urlDrilldown: 30063
   indexPatternEditor: 19123
   indexPatternFieldEditor: 34448
-  indexPatternManagement: 19165
+  dataViewManagement: 5000
```

Comment:
Is there context around why this number changed?

Reply:
Better reflection of the bundle size